source/brailleTables.py

```diff
@@ -94,12 +94,18 @@ def listTables():
 addTable("bg.ctb", _("Bulgarian 8 dot computer braille"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("ckb-g1.ctb", _("Central kurdish grade 1"))
+# Translators: The name of a braille table displayed in the
+# braille settings dialog.
 addTable("cy-cy-g1.utb", _("Welsh grade 1"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
 addTable("cy-cy-g2.ctb", _("Welsh grade 2"), contracted=True)
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("cs-comp8.utb", _("Czech 8-dots computer braille"))

```

Comment:
Please change this to "8 dot computer braille", so change the dash into a space and remove the s from dots